README.md

```diff
@@ -0,0 +1,8 @@
+# fcrepo Transform [![Build Status](https://travis-ci.org/fcrepo4-exts/fcrepo-transform.png?branch=master)](https://travis-ci.org/fcrepo4-exts/fcrepo-transform)
+
+fcrepo-transform allows you to set up and publish transformations of the RDF that composes a Fedora object's representation. You can use SPARQL for RDF-to-RDF transformations, or LDPath for RDF-to-named-fields transformations. Example uses might include presenting different views of an object's metadata to different audiences (descriptive metadata to one group, descriptive and also, live-translating from one kind of metadata to another (the way people have traditionally used XSLT with Fedora 3 objects)), or filling in the fields of an HTML form for editing metadata.
```

Comment:
Maybe splitting this up into shorter sentences? and "descriptive metadata to one group, descriptive and also..." doesn't make sense to me.